src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -72,6 +72,11 @@ export default (accessToken, userID, opts) => {
     meLoader: gravityLoader("me"),
     meBiddersLoader: gravityLoader("me/bidders"),
     meBidderPositionsLoader: gravityLoader("me/bidder_positions"),
+    bidderPositionLoader: gravityLoader(
```

Comment:
Since there is a corresponding `bidder_positions_endpoint` in gravity that is _not_ scoped to `me`, maybe we want to name this `meBidderPositionLoader` (different from the one above because it's singular)